phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexScrutinyToolIT.java

```diff
@@ -172,6 +173,14 @@
         protected long getCounterValue(Counters counters, Enum<PhoenixScrutinyJobCounters> counter) {
             return counters.findCounter(counter).getValue();
         }
+
+        protected int countRows(Connection conn, String tableFullName) throws SQLException {
+            ResultSet count = conn.createStatement().executeQuery("select count(*) from " + tableFullName);
```

Comment:
Curious: what happens if the count is way too large?

Reply:
This is integration test. It will not be large.